tools/importers/onnx/lib/onnx_converters.py

```diff
@@ -1279,11 +1279,14 @@ def get_attributes(self, attrs: Attributes):
         
     def get_output_shapes(self):
         # now compute the output size
-        channel = 0 
         if len(self.node.input_shapes) > 2:
             # third input is the output shape
+            input_shape = self.node.input_shapes[2][0]
+            if len(input_shape) == 1:
+                # normalize the shape
+                input_shape = (input_shape[0], 1)
```

Comment:
so you are making it two dimensional?  I wonder if we should just fix the ELL implementation of FullyConnectedNode so that one dimensional also works?  I assume the implementation of that should turn into a Vector x Matrix multiplication instead of Matrix x Matrix ?

Reply:
The issue is that, ONNX outputs 2-dimensional Gemm, and for example `Flatten` in the model afterwards has `axis=1`. If it is not 2-dimensional `Flatten` will fail.

Reply:
this is PyTorch's ONNX converter for `.view` which is needed for fc (fc implemented using `Linear(x.view(n_b, -1))`):

```python
            if self_sizes and len(size) == 2 and self_sizes[0] == size[0]:
                return g.op("Flatten", self, axis_i=1)
```

Reply:
I've tested this change and ran into some other problem.  For example, with this change the [mnist test model ](https://github.com/microsoft/ell-test-models/tree/master/models/mnist) fails to import properly...

Reply:
I will look at this and make sure mnist test model works first then submit another PR.